broker-core/src/main/java/org/apache/qpid/server/security/auth/manager/ManagedUser.java

```diff
@@ -103,13 +106,34 @@ public String getPassword()
     @Override
     public void setPassword(final String password)
     {
-        setAttributes(Collections.<String, Object>singletonMap(User.PASSWORD, password));
+        setAttributes(Collections.singletonMap(User.PASSWORD, password));
     }
 
     @Override
     protected void logOperation(final String operation)
     {
-        ((Container) _authenticationManager.getParent()).getEventLogger().message(AuthenticationProviderMessages.OPERATION(operation));
+        ((Container<?>) _authenticationManager.getParent()).getEventLogger()
+            .message(AuthenticationProviderMessages.OPERATION(operation));
+    }
+
+    @Override
+    protected void logCreated(final Map<String, Object> attributes, final Outcome outcome)
+    {
+        ((Container<?>) _authenticationManager.getParent()).getEventLogger()
+            .message(UserMessages.CREATE(getName(), "SUCCESS", String.valueOf(getActualAttributes())));
```

Comment:
The outcome should be used instead of the "SUCCESS" string to see if operation was successful.